src/transformers/integrations.py

```diff
@@ -932,41 +932,38 @@ def __del__(self):
 
 
 class NeptuneCallback(TrainerCallback):
-    """
+    """TrainerCallback that sends the logs to [Neptune](https://neptune.ai).
     TODO: NPT-12189 - Update the docstring
```

Comment:
Let's remove all these "TODO: NPT-12189"

Reply:
I was thinking of keeping them there until I'm done, so I can find them more easily while working on them :) If that's ok

Reply:
Sure :wink: